install/installer/pkg/components/workspace/ide/constants.go

```diff
@@ -6,7 +6,7 @@ package ide
 
 const (
 	CodeIDEImage                = "ide/code"
-	CodeIDEImageStableVersion   = "commit-f861b9d093fc8cdc728a0179b3a0c43a9155d9b4" // stable version that will be updated manually on demand
+	CodeIDEImageStableVersion   = "commit-42cdfa2031a3c93620cfa4853c9a95647e0432e8" // stable version that will be updated manually on demand
```

Comment:
@andreafalzetti this needs to be the hash that is displayed in the results in Werft:
<img width="1608" alt="image" src="https://user-images.githubusercontent.com/29888641/212396929-06b24a0a-36eb-413f-8430-30e7ec0db590.png">

Reply:
I will update it and push again

Reply:
thanks @filiptronicek, I didn't know :)

Reply:
All good! I remember doing the exact same thing. I thing it's more a fault of DevX and the process being counter-intuitive with two different commit sources 😄